admin/template/import.list_all.tpl

```diff
@@ -44,6 +44,7 @@ var flickrUserId = '{$flickrUserId}';
       <label><input type="checkbox" name="fill_posted"> {'Post date'|translate}</label>
       <label><input type="checkbox" name="fill_description" checked="checked"> {'Description'|translate}</label>
       <label><input type="checkbox" name="fill_geotag" checked="checked"> {'Geolocalization'|translate}</label>
+      <label><input type="checkbox" name="fill_views" checked="checked"> {'View counter'|translate}</label>
```

Comment:
The new language string must be added to the `language/en_UK/plugin.lang.php` file.